Packs/Sysdig/Integrations/Sysdig/Sysdig.py

```diff
@@ -0,0 +1,353 @@
+            i = 0
+            for a in x:
+                flatten(a, name + str(i) + '_')
+                i += 1
+        else:
+            out[name[:-1]] = x
+    flatten(y)
+    return out
+
+
+def get_alerts(monitor, url):
+    sdclient = SdMonitorClient(token=monitor, sdc_url=url)
+    ok, res = sdclient.get_alerts()
+    if not ok:
+        return res
+    results = []
+    for alert in res['alerts']:
+        results.append(alert)
+    readable_output = tableToMarkdown('Sysdig Alerts', results, removeNull=True)
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Sysdig.Alerts',
+        outputs_key_field='Alerts',
+        outputs=results,
+        raw_response=res)
+
+
+def get_metrics(monitor, url):
+    sdclient = SdMonitorClient(token=monitor, sdc_url=url)
+    ok, res = sdclient.get_metrics()
+    if not ok:
+        return res
+    results = []
+    for metric_id, metric in res.items():
+        results.append(metric)
+    readable_output = tableToMarkdown('Sysdig Metrics', results, removeNull=True)
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Sysdig.Metrics',
+        outputs_key_field='Metrics',
+        outputs=results,
+        raw_response=res)
+
+
+def get_events(args, monitor, url):
+    sdclient = SdMonitorClient(token=monitor, sdc_url=url)
+    eventNameInput = args['eventName']
+    results = []
+    if eventNameInput != 'All':
+        ok, res = sdclient.get_events(limit=demisto.args()['limit'], name=eventNameInput)
+    else:
+        ok, res = sdclient.get_events(limit=demisto.args()['limit'])
+    if not ok:
+        return res
+    for item in res['events']:
+        createdOn = time_convert(item['createdOn'])
+        eventDescription = item['description']
+        eventId = item['id']
+        eventName = item['name']
+        eventScope = item['scope']
+        eventSeverity = item['severity']
+        eventSource = item['source']
+        timestamp = time_convert(item['timestamp'])
+        eventType = item['type']
+        eventVersion = item['version']
+        if eventNameInput != 'All' and eventNameInput != eventName:
+            continue
+        results.append({'CreatedOn': createdOn,
+                        'EventDescription': eventDescription,
+                        'EventId': eventId,
+                        'EventName': eventName,
+                        'EventScope': eventScope,
+                        'EventSeverity': eventSeverity,
+                        'EventSource': eventSource,
+                        'Timestamp': timestamp,
+                        'EventType': eventType,
+                        'EventVersion': eventVersion
+                        })
+    readable_output = tableToMarkdown('Container Events', results, removeNull=True)
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Sysdig.Events',
+        outputs_key_field='Events',
```

Comment:
looking at the properties of event, I understand that the key field is EventId
```suggestion
        outputs_key_field='EventId',
```